webpack.config.babel.js

```diff
@@ -1,230 +0,0 @@
-          use: [
-            {
-              loader: 'css-loader',
-              options: {
-                importLoaders: 1,
-                minimize: true,
-                modules: useCssModules,
-                localIdentName: '[local]-[hash:base64:5]',
-              },
-            },
-            {
-              loader: 'postcss-loader',
-              options: {
-                plugins: postCssPlugins,
-              },
-            },
-            {
-              loader: 'sass-loader',
-              options: sassOptions,
-            },
-          ],
-        }),
-      },
-      {
-        test: /\.css$/,
-        use: ExtractTextPlugin.extract({
-          fallback: 'style-loader',
-          use: [
-            {
-              loader: 'css-loader',
-              options: {
-                importLoaders: 1,
-                minimize: true,
-              },
-            },
-            {
-              loader: 'postcss-loader',
-              options: {
-                plugins: postCssPlugins,
-              },
-            },
-          ],
-        }),
-      },
-      {
-        test: /\.(jpg|png|svg|mp4)$/,
-        exclude: /node_modules/,
-        use: [
-          {
-            loader: 'file-loader',
-            query: {
-              limit: 10000,
-              name: '[name]_[hash].[ext]',
-            },
-          },
-        ],
-      },
-      {
-        test: /favicon\.ico$/,
-        use: [
-          {
-            loader: 'file-loader',
-            query: {
-              name: '[name].[ext]',
-            },
-          },
-        ],
-      },
-      {
-        test: /\.md$/,
-        use: ['raw-loader'],
-      },
-    ],
-  },
-
-  plugins: [
-    new WrapperPlugin({
-      test: /\.css$/,
-      header: licenseHeader,
-    }),
-    new ExtractTextPlugin({
-      filename: `[name]${isProduction ? '_[contenthash]' : ''}.css`,
-      allChunks: true,
-    }),
-  ],
-
-  devServer: {
-    host: '0.0.0.0',
-    disableHostCheck: true,
-    historyApiFallback: {
-      index: 'index.html',
-    },
-  },
-};
-
-config.plugins.push(
-  new webpack.DefinePlugin({
-    'process.env': {
-      BPK_BUILT_AT,
-    },
-  }),
-);
-
-if (GOOGLE_MAPS_API_KEY) {
-  config.plugins.push(
-    new webpack.DefinePlugin({
-      'process.env': {
-        GOOGLE_MAPS_API_KEY: JSON.stringify(GOOGLE_MAPS_API_KEY),
-      },
-    }),
-  );
-}
-
-if (isProduction) {
-  config.plugins.push(
-    new StaticSiteGeneratorPlugin({
-      entry: 'docs',
-      paths: staticSiteGeneratorConfig.paths,
-      locals: staticSiteGeneratorConfig,
-    }),
-    new webpack.DefinePlugin({
-      'process.env': {
-        NODE_ENV: JSON.stringify('production'),
-      },
-    }),
-    new webpack.optimize.OccurrenceOrderPlugin(),
-    new webpack.optimize.UglifyJsPlugin({
-      compress: {
-        unused: true,
-        dead_code: true,
-        warnings: false,
-      },
-    }),
-  );
-}
-
-export default config;
```

Comment:
I guess this is no longer needed at all because it was used by the docs site?

Reply:
I believe so, as you had also removed this and looking at the way builds are done storybook uses the version in the .storybook folder